src/codec_aom.c

```diff
@@ -972,8 +972,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
             bps = 16;
         }
         aomImage.bps = bps;
-        aomImage.x_chroma_shift = alpha ? 1 : codec->internal->formatInfo.chromaShiftX;
-        aomImage.y_chroma_shift = alpha ? 1 : codec->internal->formatInfo.chromaShiftY;
+        // Monochrome is handled below. Use 4:2:0 for now.
```

Comment:
Delete this comment?

Reply:
Done.
When I first read that piece of code, I did not get why chroma_shift was set to 1 for the alpha plane, hence the hint to look below.

Reply:
Some of the fields of `aomImage` that we set don't really matter, in the sense that the libaom encoder actually ignores them. (I filed https://crbug.com/aomedia/3419 to document this better.) We can continue to set these fields to reasonable values, and after we have verified that the libaom encoder ignores these fields, we can re-evaluate if we want to remove these assignments.